test/source/TestAlgorithm.cpp

```diff
@@ -2339,6 +2341,48 @@ int TestAlgorithm()
 		}
 	}
 
+	// issue #92
+	// disable in MSVC2013 because eastl::is_copy_constructible is always true...
```

Comment:
I gave up trying to find a workaround and instead disabled this test for MSVC 2013.